### ISSUE TITLE
[console] setting empty area for an org isnt persisted in the ldap

### ISSUE THREAD

**Issue by landryb** [georchestra/georchestra#2471]
ok that's a corner case - setting an empty area for a given org does a PUT with a json containing `cities:[]` - that returns a 200 code, but the ldap isnt updated - i see the operation in the log, but the DIT isnt updated for some reason:
```
Feb  7 14:21:30 demo slapd[15919]: conn=31892 op=1 MOD dn="cn=CRAIG,ou=orgs,dc=craig,dc=fr"
Feb  7 14:21:30 demo slapd[15919]: conn=31892 op=1 MOD attr=description
Feb  7 14:21:30 demo slapd[15919]: conn=31892 op=1 RESULT tag=103 err=0 text=

$ $ldapsearch -x -b ou=orgs,dc=craig,dc=fr ou=CRAIG description
dn: cn=CRAIG,ou=orgs,dc=craig,dc=fr
description: 03155
```

I think it's because of this test : https://github.com/georchestra/georchestra/blob/master/console/src/main/java/org/georchestra/console/ds/OrgsDao.java#L296

i agree it's a fringe case, but the UI shouldnt let the admin believe that the change was persisted. Imo, the test should be removed, thus setting an empty `description` attribute ? @cmangeat ?

**Comment by landryb** [comment]
Fwiw, fixing this issue is 'easy', its just a question of removing the test and .. actually test that it does the right thing, adding a specific test for this case ?

**Comment by fvanderbiest** [comment]
Yes :-)

**Comment by RemiDesgrange** [comment]
@landryb can you point the line against a proper commit instead of master ? right now line 296 is a comment on master :-) thx

**Comment by landryb** [comment]
i *think* at the time it was the `if (org.getCities() != null)` test in https://github.com/georchestra/georchestra/blob/19.04/console/src/main/java/org/georchestra/console/ds/OrgsDao.java#L287

**Comment by landryb** [comment]
which might get fixed as a side-effect of #2868 ?

**Comment by landryb** [comment]
or if cities isnt mandatory, at least there should be an empty description attribute set into ldap, instead of just ignoring the change :)
